tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_fillbug.py

```diff
@@ -0,0 +1,34 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+import time
+from gaiatest.apps.bzlite.app import BugzillaLiteStage
+
+
+class TestFillBug(GaiaTestCase):
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.test_app = {
```

Comment:
If I will use install_app, I think i should import something? I couldn't find it in HomeScreen App or regions so there are any other way to search in ?